build/configs/rtl8730e/rtl8730e_external.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+###########################################################################
+#
+# Copyright 2023 Samsung Electronics All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+#
+###########################################################################
+# rtl8730e_download.sh
+
+TOOL_PATH=${TOP_PATH}/build/tools/amebasmart
+IMG_TOOL_PATH=${TOOL_PATH}/image_tool
+
+OS_PATH=${TOP_PATH}/os
+CONFIG=${OS_PATH}/.config
+source ${CONFIG}
+APP_NUM=0
+
+function pre_download()
+{
+	source ${TOP_PATH}/os/.bininfo
+	if test -f "${BIN_PATH}/rtk_ext_flash_data.bin"; then
+		cp -p ${BIN_PATH}/rtk_ext_flash_data.bin ${IMG_TOOL_PATH}/rtk_ext_flash_data.bin
```

Comment:
rtk_ext_flash_data.bin should be written in the board_metadata.txt.
https://github.com/Samsung/TizenRT/blob/master/build/configs/rtl8730e/board_metadata.txt
And we should use that data.